src/Entries/Collection.php

```diff
@@ -423,6 +424,7 @@ public function fileData()
                 'past' => $this->pastDateBehavior,
                 'future' => $this->futureDateBehavior,
             ],
+            'hidden' => $this->hidden,
```

Comment:
Can you make `hidden` only get written to the YAML when it's `true`?

`'hidden' => $this->hidden ? true : null` should do it.

Reply:
Ah that's how you do it. I spent a good couple minutes trying to figure that out 👍